signing_clients/apps.py

```diff
@@ -302,19 +317,26 @@ def make_signed(self, signature, outpath=None):
         if os.path.exists(outpath):
             raise IOError("File already exists: %s" % outpath)
 
+        sigpath = sigpath  or signature.filename
```

Comment:
`filename` on the signature defaults to `None` (i think), do you need to worry about `signature.filename` ever being `None`?